gcp-jobs/update-legal-filings/src/update_legal_filings/worker.py

```diff
@@ -334,3 +336,167 @@ def update_business_nos():  # pylint: disable=redefined-outer-name
+        "BC1411665", "BC1409970", "BC1409968", "BC1409966", "BC1403023", "BC1402458", "BC1402422", "BC1396800",
+        "BC1396795", "BC1396177", "BC1396133", "BC1395749", "BC1393563", "BC1392185", "BC1391097", "BC1390906",
+        "BC1255957", "BC1387965", "BC1387943", "BC1387232", "BC1386102", "BC1385653", "BC1385498", "BC1385337",
+        "BC1384652", "BC1381964", "BC1379279", "BC1374932", "BC1373942", "BC1373092", "BC1372867", "BC1372240",
+        "BC1371754", "BC1371596", "BC1363995", "BC1357406", "BC1354255", "BC1349489", "BC1349238", "BC1347809",
+        "BC1345597", "BC1342762", "BC1342086", "BC1336861", "BC1331964", "BC1324998", "BC1324894", "BC1324889",
+        "BC1324221", "BC1321272", "BC1314465", "BC1313713", "BC1313658", "BC1313261", "BC1310531", "BC1310221",
+        "BC1309930", "BC1309597", "BC1308329", "BC1186381", "BC1306000", "BC1304018", "BC1303233", "BC1302541",
+        "BC1302455", "BC1294238", "BC1294095", "BC1292965", "BC1291871", "BC1280573", "BC1278342", "BC1268600",
+        "BC1265645", "BC1263326", "BC1263195", "BC1260267", "BC1281607", "BC1422277", "BC1403939", "BC1405285"]
+    return businesses
+
+
+def check_ben_to_bc_filings(token: str) -> list[dict]:
+    """Check for new filings in COLIN."""
+    legal_url = current_app.config["LEGAL_API_URL"] + "/businesses"
+    colin_url = current_app.config["COLIN_URL"]
+    colin_events = []
+    businesses = _get_ben_to_bc_identifiers()
+    for identifier in businesses:
+        # Get the last colin event id for the identifier
+        response = requests.get(
+            f"{legal_url}/internal/last-event-id/{identifier}",
+            headers={"Content-Type": CONTENT_TYPE_JSON, "Authorization": f"Bearer {token}"},
+            timeout=AccountService.timeout)
+        last_event_id = response.json()["maxId"]
+
+        # check if there are filings to send to legal
+        colin_identifier = identifier[2:]
+        response = requests.get(
+            f"{colin_url}/event/corp_num/{colin_identifier}/{last_event_id}",
+            headers={"Content-Type": CONTENT_TYPE_JSON, "Authorization": f"Bearer {token}"},
+            timeout=AccountService.timeout
+        )
+        if response.status_code != HTTPStatus.OK:
+            current_app.logger.error("legal-updater failed to get filings from colin-api.")
+            raise Exception  # pylint: disable=broad-exception-raised
+        event_info = dict(response.json())
+        events = event_info.get("events")
+        for event in events:
+            # None filing_typ_cd found in 'BC1294238', 'BC1265645', 'BC1263326', 'BC1263195' without filings
+            if event["filing_typ_cd"] not in ["COGS1", None]:
+                event["corp_num"] = identifier
+                colin_events.append(event)
+
+    return colin_events
+
+
+def update_ben_to_bc():  # pylint: disable=redefined-outer-name, too-many-branches
```

Comment:
not calling this function from anywhere, replicating this code since we decided to complete migration of ben -> bc businesses after implementing officer (party) in lear